### ISSUE TITLE
libjellyfish-2.0.so.2: cannot open shared object file error

### ISSUE THREAD

**Issue by npbhavya** [gmarcais/Jellyfish#25]
So i have been trying to use jellyfish to count kmers of size 4, so i have been using the following command 
jellyfish count -s 400000 -t 32 -C -m 4 -o 4mers.txt <filename>
But i keep getting the following error libjellyfish-2.0.so.2: cannot open shared object file error. Can you help me understand this error and how to get past it

**Comment by pminx** [comment]
Thanks. export LD_LIBRARY_PATH="/usr/local/lib" then 'jellyfish' worked for me as well. No more libjellyfish-2.0.so.2 error. (v2.2.7) 
Pat

**Comment by gmarcais** [comment]
I do not think that the proper way to go. Setting up `LD_LIBRARY_PATH` is last resort hack in my opinion. If you installed in `/usr/local/lib`, run as root `ldconfig` and it should work.
